e2e/pages/events/draftCaseManagementOrderEvent.page.js

```diff
@@ -20,6 +21,12 @@ module.exports = {
       keyIssues: '#schedule_keyIssues',
       partiesPositions: '#schedule_partiesPositions',
     },
+    respondentDirectionsCustom: {
```

Comment:
Why is that defined here? Shouldn't that be part of `draftDirections` fragment?

Reply:
No because only the respondentsDirections and otherDirections in CMO will have this dropdown.